static_src/test/functional/overview_quicklook.spec.js

```diff
@@ -43,7 +43,9 @@ describe('Overview page', function () {
     });
 
     it('is expanded', function () {
-      expect(quicklookElement.isExpanded()).toBe(true);
+      browser.waitUntil(function() {
+        return quicklookElement.isExpanded();
+      }, 2000, 'quicklook does not expand');
```

Comment:
So would the `expand` return a promise? And then you'd run the test after the promise completes?